Remove runpath setting, so that EasyBuild can use rpathing as expected.

Author: Jan Andre Reuter (j.reuter@fz-juelich.de)

diff --color -rui ROCmValidationSuite-rocm-6.4.1.orig/CMakeLists.txt ROCmValidationSuite-rocm-6.4.1/CMakeLists.txt
--- ROCmValidationSuite-rocm-6.4.1.orig/CMakeLists.txt	2025-04-28 22:05:34.000000000 +0200
+++ ROCmValidationSuite-rocm-6.4.1/CMakeLists.txt	2025-06-25 10:11:38.787970766 +0200
@@ -24,8 +24,6 @@
 ################################################################################
 
 cmake_minimum_required ( VERSION 3.5.0 )
-set(CMAKE_SHARED_LINKER_FLAGS_INIT "-Wl,--enable-new-dtags,--rpath,$ORIGIN:$ORIGIN/.." CACHE STRING "RUNPATH for libraries")
-set(CMAKE_EXE_LINKER_FLAGS_INIT "-Wl,--enable-new-dtags,--rpath,$ORIGIN/../lib" CACHE STRING "RUNPATH for executables")
 
 project ("rocm-validation-suite"
           VERSION 1.1.0)
